pai-management/paictl.py

```diff
@@ -163,26 +165,38 @@ def cluster_object_model_generate_service(config_path):
 
 
 
-def hadoop_ai_build(os_type = "ubuntu16.04"):
+def pai_build():
 
     parser = argparse.ArgumentParser()
     parser.add_argument('-p', '--path', required=True, help="The path of your configuration directory.")
+    parser.add_argument('-n', '--imagename', default='all', help="Build and push the target image to the registry")
```

Comment:
>--imagename [](start = 31, length = 11)

Unify style to use "-" to seperate words? For example rename to "--image-name".

Reply:
resolved.